CMakeLists.txt

```diff
@@ -55,11 +55,11 @@ include_directories(./)
 
 # build spdlog as a shared library to improve compile times
 # adding this as a SYSTEM include suppresses all the terrible warnings in spdlog
+IF (WIN32)
+include_directories(third-party/spdlog/include)
+ELSE()
 include_directories(SYSTEM third-party/spdlog/include)
-# this makes spdlog generate a shared library that we can link against
```

Comment:
I will take another look at this today, hopefully we can figure out what is going wrong here.

Reply:
https://github.com/water111/jak-project/pull/69 Is relevant